components/bintray-publish/plan.sh

```diff
@@ -1,4 +1,4 @@
-# shellcheck disable=2154
+# shellcheck disable=2148,2154,2034
```

Comment:
How did these become an issue now? Has this file never been Shellchecked before?